packages/webapp/src/containers/Finances/index.jsx

```diff
@@ -98,7 +98,7 @@ const Finances = ({ history }) => {
   };
 
   const makeTransactionsSearchableString = (transaction) =>
-    [transaction.note, transaction.typeLabel || t('SALE.FINANCES.LABOUR_EXPENSE')]
+    [transaction.note, transaction.typeLabel || t('FINANCES.TRANSACTION.LABOUR_EXPENSE')]
```

Comment:
Translations were missing. I changed the key to use the existing ones.

Reply:
Ah yes sorry I got the keys confused! I fixed it in this commit two days ago but that PR is not merged. And it actually needs the other key as well so depending on what gets merged first it will conflict again sorry 🙏 

https://github.com/LiteFarmOrg/LiteFarm/pull/2958/commits/3ccb5c64c5e57ac2e9a7577a0b82d0f8583c5e70